src/AppBundle/Entity/Event.php

```diff
@@ -398,6 +420,33 @@ public function getOrganizer()
         return $this->organizer;
     }
 
+    public function addPartnerOrganizers($partnerOrganizer)
```

Comment:
This function should be called `addPartnerOrganizer` (singular “Organizer”), but some reflection magic somewhere may require it to be called `addPartnerOrganizers`.

Reply:
Changed. Magic be dammed 🪄